models/review.go

```diff
@@ -255,3 +255,32 @@ func UpdateReview(r *Review) error {
 	}
 	return nil
 }
+
+// PullReviewersWithType represents the type used to display a review overview
+type PullReviewersWithType struct {
+	User              `xorm:"extends"`
+	Type              ReviewType
+	ReviewUpdatedUnix util.TimeStamp `xorm:"review_updated_unix"`
+}
+
+// GetReviewersByPullID gets all reviewers for a pull request with the statuses
+func GetReviewersByPullID(pullID int64) (issueReviewers map[int64]*PullReviewersWithType, err error) {
```

Comment:
Would be better to reuse it also later if it would return array with latest statuses not map, in map this could lose actual order of reviews

Reply:
Yes, you can select in reverse order and than just use map[int64]bool to keep track of users already in array. And later sort.Reverse to get in correct order

Reply:
Done.